content/ordering.ts

```diff
@@ -87,7 +87,7 @@ const MODULE_ORDERING: { [key in SectionID]: Chapter[] } = {
     },
     {
       name: 'Additional',
-      description: 'Topics not strictly necessary to know to pass Bronze.',
+      description: '',
```

Comment:
probably you will need to do a problem in one of these categories to pass Bronze ...